x-pack/plugins/triggers_actions_ui/public/types.ts

```diff
@@ -749,7 +750,7 @@ export interface AlertsTableConfigurationRegistry {
     footer: AlertTableFlyoutComponent;
   };
   sort?: SortCombinations[];
-  getRenderCellValue?: GetRenderCellValue;
+  getRenderCellValue?: RenderCellValue;
```

Comment:
Since this is not a "getter" anymore, I wonder if it would make more sense to call the option `renderCellValue` directly. @XavierM what do you say?

Reply:
yes that will be better!

Reply:
neither are very accurate imo since this is ultimately treated just as a component https://github.com/elastic/eui/blob/main/src/components/datagrid/body/cell/data_grid_cell.tsx#L82 will update though